### PR TITLE
Print all styles

### PR DESCRIPTION
**Pull request by msabramo**
1. Print available styles in several groups, one per line (textStyles, colorStyles, brightColorStyles, bgColorStyles, and bgBrightColorStyles).

2. Show available styles when invalid style provided (with a bit of extra color too for the error message).

Screen Shot 2021-09-22 at 4 02 18 AM

Screen Shot 2021-09-22 at 4 01 55 AM

### CODE REVIEW

File: cli.js [hunk 1]
```diff
@@ -32,8 +32,22 @@ const printAllStyles = () => {
 		'bgCyan',
 		'bgWhite',
 	];
+	allStyles.sliceByValue = function (startValue, endValue) {
```
Comment (sindresorhus) [inline cli.js]: It's better to accept an array as argument than to add a property to the array.

**Comment by sindresorhus** [comment]
Have a look at https://github.com/chalk/chalk/blob/main/examples/screenshot.js, which does the same thing.

**Review by sindresorhus — CHANGES_REQUESTED** [review]

File: cli.js (hunk 1 above)
Comment (msabramo) [reply cli.js]: Just updated and the code no longer uses `sliceByValue` at all.

**Comment by msabramo** [comment]
Ready for review.

**Comment by sindresorhus** [comment]
Some of the foreground texts are unreadable. This is not the case for the Chalk repo demo.

**Comment by msabramo** [comment]
> Some of the foreground texts are unreadable. This is not the case for the Chalk repo demo.

Which ones in particular?

**Comment by msabramo** [comment]
I think how good this looks and which ones are hard to see will be highly dependent on the user’s terminal color settings. 

Did you think some of the ones in my screenshot were unreadable or are they unreadable on your terminal and if so I guess a screenshot could be helpful.

**Comment by sindresorhus** [comment]
Never mind. Git had screwed up and had preserved the changes from your last commits even when I updated. The latest changes look good to me.

**Review by sindresorhus — APPROVED** [review]

**Comment by msabramo** [comment]
Ah okay. Well I was going to post this screenshot I just took that compares the output of `chalk --demo` with this PR to the chalk screenshot script. I guess it's not quite as important now, but since I already took it and it might be interesting to look at, here it is:

Screen Shot 2021-09-23 at 10 17 20 AM

**Review by Qix- — APPROVED** [review]

**Pull request merged**